src/Domain/User/User.php

```diff
@@ -82,6 +91,26 @@ private function setHashedPassword(HashedPassword $hashedPassword): void
         $this->hashedPassword = $hashedPassword;
     }
 
+    private function setCreatedAt(DateTime $createdAt): void
+    {
+        $this->createdAt = $createdAt;
+    }
+
+    private function setUpdatedAt(DateTime $updatedAt): void
```

Comment:
Thinking about remove all this setters. Also SetEmail and SetHashedPassword. Thoughts?

Reply:
I agree with you, if we manage all the changes using the apply methods it does not have sense to maintain the setters.